library/ZendService/Twitter/Twitter.php

```diff
@@ -923,15 +923,45 @@ public function statusesSample()
      * Show a single status
      *
      * @param  int $id Id of status to show
+     * @param  array $options Possible list of query options
      * @throws Http\Client\Exception\ExceptionInterface if HTTP request fails or times out
      * @throws Exception\DomainException if unable to decode JSON payload
      * @return Response
      */
-    public function statusesShow($id)
+    public function statusesShow($id, array $options = [])
     {
         $this->init();
         $path = 'statuses/show/' . $this->validInteger($id);
-        $response = $this->get($path);
+        $params = [];
+        foreach ($options as $key => $value) {
+            switch (strtolower($key)) {
+                case 'include_entities':
+                    $params['include_entities'] = (bool) $value;
+                    break;
+                case 'trim_user':
+                    if (in_array($value, [true, 'true', 't', 1, '1'])) {
```

Comment:
In each of these cases, I see no reason for us to look at multiple values. The user is passing the `$options` array, which is PHP; only test for boolean `true` or `false`: `$value = (bool) $value;`.